index.js

```diff
@@ -19,14 +19,26 @@ module.exports = {
     'no-unused-vars': ['off'],
     // Use typescript version of "no-unused-vars" rule, because the default rule doesn't recognize certain typescript
     // features like enumerations
-    '@typescript-eslint/no-unused-vars': ['warn', { argsIgnorePattern: '^_', varsIgnorePattern: '^_' }],
-    'no-use-before-define': ['error', { functions: false }],
+    '@typescript-eslint/no-unused-vars': ['warn', { varsIgnorePattern: '^_', args: 'none' }],
```

Comment:
Not sure about the varsIgnorePattern, does it mean that something that starts with `_` should not be checked?
I don't really see the use case for this.

Reply:
Yep, your interpretation is correct and I'm not sure either if we actually need this. I did not question this when adjusting the function parameter stuff 🙈

I'll remove it for now. As with all other rules: If we actually have a need for this at some point, we can always re-add it...